apps/pie-docs/src/components/divider/code/props.json

```
@@ -1,25 +1,39 @@
 {
-  "headings": ["Prop", "Options", "Description", "Default"],
+  "headings": ["Prop", "Type", "Description", "Default"],
   "rows": [
       [
           "variant",
           {
               "type": "code",
               "item": ["\"default\"", "\"inverse\""]
           },
-          "Which variant of the divider to use.",
+          "Sets the variant of the divider.",
           {
               "type": "code",
               "item": ["\"default\""]
           }
       ],
+      [
+        "label",
+        {
+          "type": "code",
+          "item":  [
+            "string"
+          ]
+        },
+        "The label text for the divider. Label is only available for the horizontal variant.",
+        {
+          "type": "code",
+          "item": ["\"\""]
+        }
+      ],
       [
         "orientation",
         {
             "type": "code",
             "item": ["\"horizontal\"", "\"vertical\""]
         },
-        "Which orientation of the divider to use.",
+        "Set the orientation of the divider.",
```

just for consistency with the above 
```suggestion
        "Sets the orientation of the divider.",
```